web/src/popovers.js

```diff
@@ -1142,7 +1142,7 @@ export function hide_all_except_sidebars(opts) {
     stream_popover.hide_stream_popover();
     stream_popover.hide_topic_popover();
     stream_popover.hide_all_messages_popover();
-    stream_popover.hide_starred_messages_popover();
+    popover_menus.get_starred_messages_popover()?.hide();
```

Comment:
`hideAll();` above should already work to hide the popover so this should just be a delete instead.

Reply:
Sure, thanks!